src/iotEdgeExplorer.ts

```diff
@@ -124,7 +124,18 @@ export class IoTEdgeExplorer extends BaseExplorer {
         for (const moduleName in modules) {
             if (modules.hasOwnProperty(moduleName)) {
                 try {
-                    const createOptions = modules[moduleName].settings.createOptions;
+                    let createOptions = modules[moduleName].settings.createOptions;
+
+                    let i = 0;
+                    while (i < Constants.CREATE_OPTIONS_MAX_CHUNKS) {

```

Comment:
Make sure it is from `01` to `07`, current logic will have `08`

Reply:
Updated